src/main/java/org/fogbowcloud/manager/core/controllers/ComputeOrdersController.java

```diff
@@ -0,0 +1,38 @@
+package org.fogbowcloud.manager.core.controllers;
+
+import java.util.List;
+
+import org.fogbowcloud.manager.core.models.orders.ComputeOrder;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@CrossOrigin
+@RestController
+@RequestMapping(value = "compute")
+public class ComputeOrdersController {
+	@RequestMapping(method = RequestMethod.POST)
+	public ResponseEntity<ComputeOrder> createCompute(@RequestBody ComputeOrder ComputeOrder) {
```

Comment:
The HTTP POST return should not be a ComputeOrder, if think that should be an OK or ERROR status (String Representation).